modules/nw-networkpolicy-optimize.adoc

```diff
@@ -2,8 +2,8 @@
 //
 // * networking/network_policy/about-network-policy.adoc
 
-[id="nw-networkpolicy-optimize_{context}"]
-= Optimizations for network policy
+[id="nw-networkpolicy-optimize-sdn_{context}"]
+= Optimizations for network policy with SDN
```

Comment:
that note
```
[NOTE]
====
====
The guidelines for efficient use of network policy rules applies to only the OpenShift SDN network plugin.
The guidelines for efficient use of network policy rules applies to only the OpenShift SDN network plugin.
====
====
```
may not be required anymore?
Also, looks like we usually `OpenShift SDN` instead of just `SDN`, so maybe we should update the title?